supertokens_python/recipe/session/utils.py

```diff
@@ -123,10 +136,15 @@ def __init__(
             [BaseRequest, str, BaseResponse],
             Union[BaseResponse, Awaitable[BaseResponse]],
         ],
+        on_invalid_claim: Callable[
+            [BaseRequest, List[Dict[str, Any]], BaseResponse],
```

Comment:
ClaimValidationError[] gets converted into List[Dict[str, Any]] because that's what the client expects:

```python
class InvalidClaimsError(SuperTokensSessionError):
    def __init__(self, msg: str, payload: List[ClaimValidationError]):
        super().__init__(msg)
        self.payload: List[Dict[str, Any]] = []
        for p in payload:
            res = (
                p.__dict__.copy()
            )  # Must be JSON serializable as it will be used in response
            if p.reason is None:
                res.pop("reason")
            self.payload.append(res)
```

Reply:
This conversion should happen in the default implementation of handling invalid claims, and not before that.

Reply:
done